.gitignore

```diff
@@ -39,6 +39,7 @@ ipython_config.py
 # Environments
 .python-version
 .env
+.envrc
```

Comment:
It looks like we now _require_ `uv` virtualenv management, so I'm hoping I can keep a local `.envrc` that helps me navigate that with my `pyenv` setup.